### PR TITLE
[WIP] Added compatibility for IntelliJ IDEA CE & Ultimate 2020.3

### PR DESCRIPTION
**Pull request by c16a**
ij_product definitions are intellij-canary and intellij-ue-canary

Signed-off-by: Chaitanya Munukutla <chaitanya.m61292@gmail.com>

# Checklist

- [ ] I have filed an issue about this change and discussed potential changes with the maintainers.
- [ ] I have received the approval from the maintainers to make this change.
- [x] This is not a stylistic, refactoring, or cleanup change.

Please note that the maintainers will not be reviewing this change until all checkboxes are ticked. See 
the [Contributions](https://github.com/bazelbuild/intellij#contributions) section in the README for more 
details.

# Discussion thread for this change

Issue number: #2102

# Description of this change

This PR adds support for IntelliJ 2020.3. CLion 2020.3 is NOT supported yet.

### CODE REVIEW

File: python/src/com/google/idea/blaze/python/issueparser/PyIssueParserProvider.java [hunk 1]
```diff
@@ -130,7 +130,8 @@ private static int rankResult(PsiFile file, GlobalSearchScope projectScope, Sdk
       if (projectScope.contains(vf)) {
         return 0;
       }
-      return PythonSdkType.isStdLib(vf, sdk) ? 2 : 1;
+      return 2;
+      // return PythonSdkType.isStdLib(vf, sdk) ? 2 : 1;
```
Comment (c16a) [inline python/src/com/google/idea/blaze/python/issueparser/PyIssueParserProvider.java:134]: @mprobst Looks like the `isStdLib` function has been removed in the latest Python plugin. Any thoughts on how to handle this? The new version doesn't seem to have any relevant alternatives.

**Comment by mistic** [comment]
@alice-ks can we take a look on this to moving it forward please?

**Comment by alice-ks** [comment]
Thank you for this pull request. We're currently also looking into 2020.3 compatibility as you might have seen from pull requests such as #2305. We just need to take some smaller steps this time as we've ran into some challenges with our internal setup.

Pull requests such as this one or #2246 will help us when we need to figure out how to fix the code broken with 2020.3. Hence, thanks for helping us to move forward here!

**Comment by mistic** [comment]
@alice-ks thanks your answer here! I've been following that plugin for the last 7 months and I have the impression that the compatibility with the intellij latest versions tend to land several weeks after the IDE release. Do you have any plans for a new strategy in order to as much as possible update the plugin around the release date of the IDE?

**Comment by AlexeyGy** [comment]
Thanks again for creating this, we have by now added support for 2020.3 in a beta release which you can now also install in IntelliJ https://github.com/bazelbuild/intellij/issues/2102 

Please do not feel like this PR was in vain! When implementing the changes, I had looked at your suggestions and found them very helpful. Unfortunately, due to the nature of the code partly living on our internal repository, we cannot merge PRs easily at the moment. We have plans to improve this in the future.

**Comment by c16a** [comment]
Thanks @AlexeyGy!

**Pull request closed without merging**